SET_OF_STANDARDS.md

```diff
@@ -0,0 +1,50 @@
+# Programming Set of Standards
+
+## Purpose
+These are primarily guidelines on contributing ideas and code to the WeLike codebase. As well, there are other helpful insights into what is expected of us as technology industry professionals.
+
+## Note
+Since ppl can’t meet at the same time, please add your coding needs, NTKs, and suggestions below. Create sections as well. If there is a suggestion that you do NOT agree with, do NOT delete it! Instead add a comment to it and tag the contributor.
+
+Interesting source - JavaScript Coding Standards
+
+## Git
+Keywords for A Commits's Subject Matter Message: feat | fix | chore | refactor | docs | style | test | perf | ci | build | revert
+
+When in doubt, refer to this helpful freeCodeCamp post! [How to Level Up Your Git Commit Messages](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)
+
+Ex. of command line entry for commit message: git commit -m "fix: fix foo to enable bar"
+
+## PRs
+Comments need to be professional. Stems from this Discord post
+Prepending every pr title with the dev’s name. Ex: “Meke/Fix Content Row”
+Agreed-upon outline for new PRs tbd. Ex can be found here
+Meke doesn’t merge code if she approves a pr
+
+## The code itself
+Adding code to file another is “Responsible” for. 
+Single or double quotes?
+
+## Deleting repos
+Please don’t! Why? This:
+Loses all previous work for everyone
+Loses prs with important details
+Generally is a no-no
+
+## Code comments
+
+## Spacing
+
+## Naming convention
+Camel casing, snake casing, pascal casing?
+Different for FE and BE?
+Meke likes to use snake case for BE names, and camel for FE
+
+## Branches
+Prepending every branch name with the dev’s name Ex: “meke/content-row-fix”
+
```

Comment:
We should probably not encourage force pushes on branches at a certain point, because that removes the commit history for that branch. I'm guilty of that from time to time, so I'd have to adjust as well. We can enforce that a few different ways, including Github settings.

Reply:
Sounds good! I look forward to you adding that sentiment when we get together to go over this.

Reply:
https://github.com/orgs/QueerGlobal/projects/3?pane=issue&itemId=32362036